config/manifests/bases/camel-k.clusterserviceversion.yaml

```diff
@@ -22,16 +22,16 @@ metadata:
     capabilities: Full Lifecycle
     categories: Integration & Delivery
     certified: "false"
-    containerImage: docker.io/apache/camel-k:1.10.0-SNAPSHOT
-    createdAt: 2022-05-31T12:18:30Z
+    containerImage: docker.io/apache/camel-k:X.Y.Z-SNAPSHOT
+    createdAt: 2022-07-05T16:41:32Z
     description: Apache Camel K is a lightweight integration platform, born on Kubernetes,
       with serverless superpowers.
-    operators.operatorframework.io/builder: operator-sdk-v1.3.0
+    operators.operatorframework.io/builder: operator-sdk-v1.16.0
```

Comment:
Do we want to change this? I'd keep it the original version unless there is a reason to change it.

Reply:
operator-sdk version in Makefile is 1.16. For what its worth, this is just a notifying property of the version used to build the CSV bundle. It does not actually have any material effect on OLM.